### DISCUSSION TITLE
Tasmota device resetting after power failure

### DISCUSSION THREAD

**Discussion by amiltonjr** [arendst/Tasmota#17478] [Q&A]
Hi there, I'm aware that Tasmota resets all of its settings whenever sequentially power failures occur. But that behavior is causing my device to remain OFF while it must be ON all the time. Is it possible to change the default settings to pre-defined ones, like my Wi-Fi SSID and password, to be ON by default, and also the device type (in my case Sonoff Pow R2)? At least it wouldn't stop working. I'm using the latest Tasmota release.

**Comment by sfromis** [comment]
The command `SetOption65` allows you to control the quick power cycle recovery feature, allowing you to reduce the options in case of you needing to recover the device.

**Reply by amiltonjr** [reply]
I just set it to **1**. Let's see how it behaves. Thank you!

```
SetOption65
Device recovery using fast power cycle detection
0 = enabled (default)
1 = disabled
(https://tasmota.github.io/docs/Device-Recovery/#fast-power-cycle-device-recovery)
```
